.github/workflows/promote.yml

```diff
@@ -1,48 +0,0 @@
-name: Promote
```

Comment:
this promote workflow was never used for crossplane-runtime, and therefore safe to delete?

Reply:
Yeah, exactly. I can't think of what it would have done.